src/new-client/src/Web/context.tsx

```diff
@@ -0,0 +1,4 @@
+import { createContext } from "react"
+export const TornOutTileContext = createContext({ isTornOut: false })
```

Comment:
Perhaps a more generic context could be used so we remove the duplication - `SectionTornOutContext` or similar

Reply:
Fixed, you can find the implementation at `@/components/tearOutContext`